lib/luajit2/src/vm_ppc.dasc

```diff
@@ -0,0 +1,6523 @@
+|// Low-level VM code for PowerPC 32 bit or 32on64 bit mode.
```

Comment:
If I'm not wrong, this PR contains only for PowerPC VM code. There is no ppc64le (PowerPC 64bit little endian) support.
ppc64le support should be proceeded (or stucked) here: https://github.com/LuaJIT/LuaJIT/pull/54